src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
```

Comment:
**[optional]**
we can equally destructure at the argument level like this `const Button = ({name}) => {`